autoload/nvimgdb.vim

```diff
@@ -5,13 +5,24 @@ lua gdb = require("gdb.app")
 
 
 function! s:GdbKill()
+  if &hidden
```

Comment:
Could you please also put a comment here briefly explaining why doing so? For instance: prevent "ghost" noname buffers left when 'hidden' was set.